pre_commit/hook_executor.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
```

Comment:
Yes, that was also my main concern. I don't really like adding those classes.
Glad you suggest a different approach.
Are you ok if I drop all the HookExecuter classes, but create a LocalRepository subclass of Repository ?